app/templates/resources/add.html

```diff
@@ -0,0 +1,133 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui grid container">
+        <div class="eight wide computer sixteen wide mobile centered column">
+            <h2 class="ui dividing header">Add a resource</h2>
+
+            {% set flashes = {
+                'error':   get_flashed_messages(category_filter=['form-error']),
+                'warning': get_flashed_messages(category_filter=['form-check-email']),
```

Comment:
Copied this form from register form: https://github.com/hack4impact/women-veterans-rock/blob/4ec43a736dbfec3ea671ce8256b7a62e5a066681/app/templates/account/register.html